packages/botonic-cli/src/commands/deploy.ts

```diff
@@ -83,10 +85,10 @@ Uploading...
         name: 'signupConfirmation',
         message:
           'You need to login before deploying your bot.\nDo you have a Hubtype account already?',
-        choices: choices
+        choices
       }
     ]).then((inp: any) => {
-      if (inp.signupConfirmation == choices[1]) return this.askLogin()
+      if (inp.signupConfirmation === choices[1]) return this.askLogin()
```

Comment:
Personal preference but I don't really like inline if/else, you could replace this whole block with ternary.